src/gtk/infobar.cpp

```diff
@@ -196,7 +196,9 @@ void wxInfoBar::ShowMessage(const wxString& msg, int flags)
     if ( wxGTKImpl::ConvertMessageTypeFromWX(flags, &type) )
         gtk_info_bar_set_message_type(GTK_INFO_BAR(m_widget), type);
     gtk_label_set_text(GTK_LABEL(m_impl->m_label), wxGTK_CONV(msg));
-
+    gtk_label_set_line_wrap(GTK_LABEL(m_impl->m_label), TRUE );
```

Comment:
```suggestion
    gtk_label_set_line_wrap(GTK_LABEL(m_impl->m_label), TRUE);
```
You can use spaces or not, but not use them asymmetrically, this is really too weird.